recipes/gz-sim-yarp-plugins/activate.msh

```diff
@@ -0,0 +1,5 @@
+if_(is_set("COMSPEC")).then_([
+	sys.list_append("GZ_SIM_SYSTEM_PLUGIN_PATH", path.join(env("CONDA_PREFIX"), "install\\bin"))
```

Comment:
Are you sure about this? I would have expected a different value:

```suggestion
	sys.list_append("GZ_SIM_SYSTEM_PLUGIN_PATH", path.join(env("CONDA_PREFIX"), "Library\\bin"))
```

Reply:
I took that from https://github.com/robotology/gz-sim-yarp-plugins/actions/runs/8476260204/job/23225530006#step:13:60 but maybe I'm missing something